README.md

```diff
@@ -0,0 +1,73 @@
+# CodeX Surveys
+
+A simple widget to collect users' feedback.
+
+<img width="648" alt="surveys" src="https://user-images.githubusercontent.com/26390210/213229181-5bc1e5b2-65b5-4ee6-80f6-1bfcbc9b45ea.png">
+
+## How to use Surveys
+
+1.  Create Notion [integration](https://www.notion.so/my-integrations)
+2.  Create Notion [database](https://www.notion.so/help/guides/creating-a-database)
+3.  Add connection to database
+    Right menu > Connections > Add connection
+
+4.  Install the package
+
+```bash
+  yarn add @codexteam/surveys
+```
+
+5.  Add widget to your page
+
+```javascript
+import { createWidget } from '@codexteam/surveys';
+
+createWidget({
```

Comment:
From API perspective it would be more clear if initialisation looked like this
```
CodeXSurveys.init({
  ...
})
```
(inspired by [google auth](https://developers.google.com/identity/protocols/oauth2/javascript-implicit-flow) and [apple auth](https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_js/configuring_your_webpage_for_sign_in_with_apple) js libs)

Don't we want to rename the function?

Reply:
could we use a class as entry point?

```js
const widget = new CodeXSurveys({..})
```